common/culture/innovations/wc_cultural_maa_innovations.txt

```diff
@@ -38,10 +97,876 @@ innovation_taming_the_skies = {
 }
+
+### MAA TECHS
+
+####Tribal
+innovation_archmage = {
+	group = culture_group_regional
+	culture_era = culture_era_tribal
+	icon = @archmage
+
+	potential = {
+		AND = {
+			OR = {
+				this.culture_group = culture_group:human_group
+				this = culture:blood_elf
+				this = culture:high_elf
+			}
+			NOT = {
+				this = culture:pirate_culture
+				this = culture:wastewander	
+			}
+		}
+	}
+
+	unlock_maa = archmage
+
+	flag = global_maa
+}
+
+
+innovation_blademaster = {
+	group = culture_group_regional
+	culture_era = culture_era_tribal
+	icon = @horde_special
+
+	potential = {
+		this = culture:burning_blade
+	}
+
+	unlock_maa = blademaster
+
+	flag = global_maa
+}
+
+innovation_buccaneer = {
+	group = culture_group_regional
+	culture_era = culture_era_tribal
+	icon = @buccaneer
+
+	potential = {
+		OR = {
+			this = culture:pirate_culture
+			this = culture:blackwater
+		}
+	}
+
+	unlock_maa = buccaneer
+
+	flag = global_maa
+}
+
+innovation_brute = {
+	group = culture_group_regional
+	culture_era = culture_era_tribal
+	icon = @brute
+
+	potential = {
+		this.culture_group = culture_group:gnoll_group
+	}
+
+	unlock_maa = brute
+
+	flag = global_maa
+}
+
+innovation_candleholder = {
+	group = culture_group_regional
+	culture_era = culture_era_tribal
+	icon = @candleholder
+
+	potential = {
+		this.culture_group = culture_group:kobold_group
+	}
+
+	unlock_maa = candleholder
+
+	flag = global_maa
+}
+
+innovation_dervishes = {
+	group = culture_group_regional
+	culture_era = culture_era_tribal
+	icon = @dervishes
+
+	potential = {
+		this = culture:wastewander
+	}
+
+	unlock_maa = dervishes
+
+	flag = global_maa
+}
+
+innovation_dire_troll = {
+	group = culture_group_regional
+	culture_era = culture_era_tribal
+	icon = @dire
+
+	potential = {
+		this.culture_group = culture_group:troll_group
+	}
+
+	unlock_maa = dire_troll
+
+	flag = global_maa
+}
+
+innovation_dragon = {
+	group = culture_group_regional
+	culture_era = culture_era_tribal
+	icon = @flying
+
+	potential = {
+		OR = {
+			this.culture_group = culture_group:dragon_group
+			this = culture:dragonmaw
+		}
+	}
+
+	unlock_maa = dragon
+
+	flag = global_maa
+}
+
+innovation_gladiator = {
+	group = culture_group_regional
+	culture_era = culture_era_tribal
+	icon = @horde_special
+
+	potential = {
+		this = culture:shattered_hand
+	}
+
+	unlock_maa = gladiator
+
+	flag = global_maa
+}
+
+innovation_troll_ranged = {
+	group = culture_group_regional
+	culture_era = culture_era_tribal
+	icon = @headhunter
+
+	potential = {
+		this.culture_group = culture_group:troll_group
+	}
+
+	unlock_maa = headhunter
+
+	flag = global_maa
+}
+
+innovation_hexer = {
+	group = culture_group_regional
+	culture_era = culture_era_tribal
+	icon = @hexer
+
+	potential = {
+		OR = {
+			this.culture_group = culture_group:troll_group
+			this.culture_group = culture_group:harpy_group
+			this = culture:saurok
+		}
+	}
+
+	unlock_maa = hexer
+
+	flag = global_maa
+} 
+
+innovation_marauder = {
```

Comment:
There's one thing I believe is worth discussing.
I would make cultural MaAs about the same price as generic MaAs.
Why?
Sometimes you go cheaper MaAs like light infantry because you don't have money for high infantry.
It will be very sad if players are forced to have generic MaAs because cultural MaAs cost too much.
<details>
<summary>Click to expand</summary>

![image](https://user-images.githubusercontent.com/46384992/133159443-61b7800d-fb13-4c87-8a27-af3fbdc4ea1e.png)

</details>

Reply:
I think it is the point, they cost a lot for a reason,  there should be a choice presented,  better army or something else. I think every charecter starts with enough to buy at least 1 special maa. If you have to make due with basic maa, the special will feel like progression as it will serve as a direct upgrade. If they are relatively the same price as basic there is really no point imo

Reply:
I agree with zumbak on this thing again. Sometimes there simply are no other units then the cultural MaA we made, so giving them the choice between generic and cultural makes no sense to me...

Reply:
Also it not that difficult to get the resources to get these maa's, they aren't wonder levels of cost, and their upkeep is mild so I do not think it is too bad

Reply:
> Also it not that difficult to get the resources to get these maa's, they aren't wonder levels of cost, and their upkeep is mild so I do not think it is too bad

On my screenshot, it's more expensive 40%.

Reply:
Yes I know, I highly disagree that they should be around the same cost, I genuinely think it is not too expensive and it is not out of reach for anyone. 40% sounds like a lot but in the context of around 100-200 prestige/gold its not that much.

Reply:
> 40% sounds like a lot but in the context of around 100-200 prestige/gold its not that much.

I'm fine if it's 20%.

Reply:
Should we put this to a vote in dev? Maybe you make a range of %, or you are deadset on 20%?

Reply:
> Should we put this to a vote in dev?

You can, yes, but explain the point.

> Maybe you make a range of %, or you are deadset on 20%?

I mean 20% max.

Reply:
@zumbak04 from what I know, this should be fixed now, right? Can we mark this conversation as resolved then?